Assets/Scripts/PositionTracker.cs

```diff
@@ -69,6 +73,7 @@ private void FinishRace(Bullseye bullseyeObj, Vector3 bullseyeHitPoint)
         score.SetAccuracyScore(bullseyeObj.GetScore(bullseyeHitPoint));
 
         // Compute time score here
+        score.SetTimeScore((int)(Time.time - startTime));
```

Comment:
Would it be possible to have the time score be a "positive" number? as in the greater the time score the greater the final score?

For example we could have the first racer to finish always having a set maximum score and then decrease the score of the other racers based on the time they take compared to that initial time?